### PR TITLE
Jumping

### PR DESCRIPTION
**Pull request by Shatur**

### CODE REVIEW

File: networked_controller.gd [hunk 1]
```diff
@@ -53,20 +58,23 @@ func controller_process(delta: float, db: DataBuffer):
...
-	get_parent().step_body(delta, Vector3(input_direction.x, 0.0, input_direction.y))
+	get_parent().step_body(delta, Vector3(input_direction.x, 0.0, input_direction.y), is_jumping)
 
 
 func count_input_size(inputs: DataBuffer) -> int:
 	# Count the input buffer size.
 	var size: int = 0
 	size += inputs.get_bool_size()
+	size += inputs.get_bool_size()
+	inputs.skip_bool()
```
Comment (AndreaCatania) [inline networked_controller.gd:76]: I would invert this, so to have:
```
size += inputs.get_bool_size()
inputs.skip_bool()

size += inputs.get_bool_size()
if inputs.read_bool():
```

It should be cleaner.

**Review by AndreaCatania — CHANGES_REQUESTED** [review]
Looks good, just a minor thing to change. Regarding the debug lines, did you remove it because they are not yet implemented in Godot 4?

**Comment by Shatur** [comment]
> Regarding the debug lines, did you remove it because they are not yet implemented in Godot 4?

Yep. It will also make the example more simple. We could restore it later because we use git now :)

File: networked_controller.gd (hunk 1 above)
Comment (Shatur) [reply networked_controller.gd:76]: Agree!

**Review by AndreaCatania — APPROVED** [review]

**Comment by AndreaCatania** [comment]
Amazing, thanks!!

**Pull request merged**